pyha_analyzer/chunking_methods/audio_splitter.py

```diff
@@ -0,0 +1,94 @@
+""" Splits longer audio files into smaller ones """
+
+import os
+import pandas as pd
+import torch
+import torchaudio
+
+CONFIG = {
+    "metadata_csv": "annotations_chunked.csv",
+    "metadata_output": "annotations_split.csv",
+
+    "audio_dir": "input",
+    "sample_rate": "error", # Only use if input format is pt
+    "output_dir": "output",
+    "output_format": "flac", # Supports "wav" or "pt"
```

Comment:
.pt files would only be used for raw chunking, otherwise they're generated during runtime - what does this pipeline feed into (or intended)?

Reply:
makes sense, removing it